content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,104 @@ weight: 5
+2. Releasing the lever is to have the foreknowledge that some people might be killed.
 
-### Moral Consistency
+Knowingly risking some deaths is morally preferable to knowingly killing with intent.
 
-Trying to trip vegans up by exposing real or imagined inconsistencies is a common avenue of argument against veganism. The reality is that vegans (being human) are typically not perfect. Vegans are subject to the constraints of time, money, knowledge, and availability of resources amongst a million other factors. We quoted the definition of veganism in our quick response but it's so important it bears repeating: 
+Humans hold a lever which results in the intentional deaths of millions of animals. It's time to let go of the lever; the foreknowledge that some animals will probably die as a result of crop farming is not an argument against veganism.
 
-> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals  for food, clothing or any other purpose.  
+> Vegans do not accept the deaths of some animals to produce our foods of preference. While vegans have foreknowledge that this may occur it is not the intent and, wherever possible, steps are taken to reduce and mitigate this harm. 
+> 
+>  There is considerable moral difference between slaughtering animals with intent and deaths that occur as an unintentional side effect of crop farming, even where we have foreknowledge the deaths might occur. 
+
+
+### Argument #3 *Definition of Veganism*
+
+The definition of veganism is in our Quick Response but it's so important it's worth repeating: 
+
+> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals for food, clothing or any other purpose.  
 > - [The Vegan Society - Definition of Veganism](https://www.vegansociety.com/go-vegan/definition-veganism)
 
-Each person's 'possible and practicable' is a line they have to draw for themselves. It is not necessary to be perfect to be vegan. 
+#### * What does *possible and practicable* mean?
+
+This is not an excuse to have a steak when there isn't a vegan option at the restaurant. Eating a boring salad is a very minor discomfort by comparison with the horrors that animals face in slaughterhouses. 
+
+Nor is it an excuse to not learn those pesky [E-numbers] ({{< ref "../shopping/e-numbers" >}}). The mistreatment of animals does not compare to committing these to memory, or carrying around the TalkVeganToMe App.
+
+
+
+> *Possible and practicable* is NOT a line vegans draw for themselves to mitigate some perceived inconvenience. This would reduce veganism to some form of moral relativism.   
+> Rather, it is a reference to the threshold a ‘rational person’ would set; ‘all other things being equal’. 
```

Comment:
As much as I agree philosophically, I think this teeters too close to the edge of transgressing [mission statement #6](https://github.com/talkvegantome/talkvegan-hugo)

> 6 Not to engage in the gatekeeping of veganism (no-one can take away your V card)

Or at the very least provides a stepping stone for other people to do so. 

Can we do without it?